test/integration/targets/route53/tasks/main.yml

```diff
@@ -9,6 +9,7 @@
 - name: Test basics (new zone, A and AAAA records)
   module_defaults:
     group/aws:
+      debug_botocore_endpoint_logs: True
```

Comment:
Can this be safely enabled for all AWS tasks in our tests?

If so, ansible-test could be extended so cloud plugins could set module defaults at the play level instead.

Reply:
To extend ansible-test to enable setting defaults at the task level, you'll need to add another method to the base class that can return module defaults:

https://github.com/ansible/ansible/blob/f97abc095b244038b4e4d59fb7919ba15643c230/test/runner/lib/cloud/__init__.py#L378

Then add that method to the aws.py cloud plugin to return the needed setting.

Call the method and use it when generating the playbook:

https://github.com/ansible/ansible/blob/f97abc095b244038b4e4d59fb7919ba15643c230/test/runner/lib/executor.py#L1185-L1195

Note that this won't work for script based tests since they don't use our generated playbook.